staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -55,7 +55,7 @@ const (
 // kubelet is running in the host's root mount namespace.
 type Mounter struct {
 	mounterPath string
-	withSystemd bool
+	withSystemd *bool
```

Comment:
@nixpanic cant we keep it as bool ?

Reply:
That would make it only `true` or `false`. With a pointer to a bool it gets initialized as `nil`, `hasSystemd()` can check that and run `detectSystemd()` when needed. There is only one place where `withSystemd` is replaced by `hasSystemd()`, so I thought this was rather simple and clean.

Reply:
Should be fine with ptr too.